### PR TITLE
Model card template: Move model usage instructions out of Bias section

### PR DESCRIPTION
**Pull request by NimaBoscarino**
Not sure that `### How to Get Started with the Model` belongs under the `Bias, Risks, and Limitations` section, so this PR changes it from a H3 to an H2.

### CODE REVIEW

**Comment by Wauplin** [comment]
cc-ing @EziOzoani @meg-huggingface for review on this PR

Has it been discussed on Slack btw? (if yes, I would refer to this discussion for more context, otherwise I'm fine with having only this PR)

**Review by Wauplin — APPROVED** [review]
Yep agree with you, seems like a typo.
Approved the PR but please wait to get another approval from @EziOzoani or @meg-huggingface before merging, just to be sure :)

**Review by EziOzoani — APPROVED** [review]

**Pull request merged**
